test/e2e/handler/main_test.go

```diff
@@ -50,7 +51,7 @@ var _ = BeforeSuite(func() {
 
 	testenv.Start()
 
-	By("Getting node list from cluster")
+	By("Getting worker node list from cluster")
```

Comment:
Please elaborate in the commit message why do we need both `nodes` and `allNodes`. It is not clear to me.

Reply:
Done